03 - Variables/declaring_variables.rb

```diff
@@ -0,0 +1,58 @@
+# variable start woith lower case or underscore
+# variable should have letters, numbers, underscores
+# variables should not have spaces
+# we can also snake-case formatting in bigger variables like my_favorite_fruit
+# there are 40+ reserved keyword in ruby, we should not use them as variable names like print, puts,
+
+
+age=45
+puts age
+puts age+3
+
+f_name="Haris"
```

Comment:
set proper name of the variables
like first_name and last_name

Reply:
ok I will make sure of it.